internal/services/loganalytics/log_analytics_datasource_windows_event_resource.go

```diff
@@ -77,7 +77,7 @@ func resourceLogAnalyticsDataSourceWindowsEvent() *pluginsdk.Resource {
 				Elem: &pluginsdk.Schema{
 					Type: pluginsdk.TypeString,
 					// API backend accepts event_types case-insensitively
-					ValidateFunc: validation.StringInSlice([]string{"Error", "Warning", "Information"}, false),
+					ValidateFunc: validation.StringInSlice([]string{"Error", "Warning", "Information"}, true),
```

Comment:
how come we are changing to ignore case here?

Reply:
The symptom is that there is tf diff while importing this resource.
After investigated, I found tf only allows the capital initial but tf would set it to lower case while reading resource. So it doesn't pass the property validation while using the result returned by API to import this resource. I've updated PR to remove the logic to lower case while reading. Please take another look. Thanks.